app/src/main/kotlin/at/bitfire/davdroid/ui/intro/IntroScreen.kt

```diff
@@ -78,8 +79,8 @@ fun IntroScreen(
                     .padding(horizontal = 128.dp)
                     .align(Alignment.Center)
                     .fillMaxWidth(),
-                selectedIndicatorColor = MaterialTheme.colorScheme.onPrimary,
-                unselectedIndicatorColor = MaterialTheme.colorScheme.tertiary,
```

Comment:
We could leave anything in this file as it is when we just change:

```
   selectedIndicatorColor = MaterialTheme.colorScheme.onPrimary,
   unselectedIndicatorColor = MaterialTheme.colorScheme.onPrimaryContainer,
```

What do you think of that?

In theory, normal and container colors shouldn't be mixed, but as far as I remember, `Switch` does exactly that by default with the `thumbContent` thing.